src/apps/loyalty/containers/inquiries/browser.tsx

```diff
@@ -7,22 +7,31 @@ import { artsyNetworkLayer } from "../../../../relay/config"
 import CurrentUserRoute from "../../../../relay/queries/current_user"
 import Inquiries from "./index"
 
-const sharify = require("sharify")
+import * as sharify from "sharify"
+import { ResponseLocalData } from "../../types"
+
+import * as Artsy from "../../../../components/artsy"
+
+const { CURRENT_USER, RELAY_DATA } = sharify.data as ResponseLocalData
```

Comment:
`ResponseLocalData` is an interface that describes the exact data exposed to the client in the `loyalty` app.